packages/wallets/src/features/cfd/screens/ChangePassword/InvestorPassword/MT5ChangeInvestorPasswordScreens.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { SentEmailContent } from '../../../../../components';
+import { useModal } from '../../../../../components/ModalProvider';
+import MT5ChangeInvestorPasswordInputsScreen from './MT5ChangeInvestorPasswordInputsScreen';
+import MT5ChangeInvestorPasswordSavedScreen from './MT5ChangeInvestorPasswordSavedScreen';
+
+type TChangeInvestorPasswordScreenIndex = 'emailVerification' | 'introScreen' | 'savedScreen';
+
+const MT5ChangeInvestorPasswordScreens = () => {
+    const [activeScreen, setActiveScreen] = useState<TChangeInvestorPasswordScreenIndex>('introScreen');
+    const handleClick = (nextScreen: TChangeInvestorPasswordScreenIndex) => setActiveScreen(nextScreen);
+    const { hide } = useModal();
+
+    switch (activeScreen) {
+        case 'savedScreen':
+            return <MT5ChangeInvestorPasswordSavedScreen setNextScreen={hide} />;
+        case 'emailVerification':
+            return (
+                <div className='wallets-change-password__sent-email-wrapper'>
```

Comment:
minor - this effectively uses css from ChangePassword component within MT5ChangeInvestorPasswordScreen. So breaks the BEM core principle of keeping components css isolated.

Reply:
@wojciech-deriv got your mind. I'll try to modify it soon